index.html

```diff
@@ -149,7 +149,7 @@ <h1 class="project-heading">Multi-Post Stories</h1>
         <button type="button">See Project</button>
       </div>
     <div class="project-image image-4-b"><img src="./public/Snapshoot Portfolio (3).png" alt="Project-1"></div>
-    </section>
+    </section> -->
 <!-- projects section ends here  -->
 
 <!-- About me section ends here  -->
```

Comment:
- [ ] Kindly consider removing all the old pieces of code commented. they make your project looks messy. If you need to review a previous version of your code, you can always use git history.